frontend/src/metabase/dashboard/selectors/selectors-typed.ts

```diff
@@ -5,6 +5,10 @@ export function getDashboardId(state: State) {
   return state.dashboard.dashboardId;
 }
 
+export function getDashboardBeforeEditing(state: State) {
+  return state.dashboard.isEditing;
```

Comment:
Oh, `isEditing` is a dashboard instead of a boolean 🤯

Reply:
I'm afraid you're right 😢